addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -51,7 +51,7 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
     def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kwargs):
         values = {}
         product = self.env['product.product'].browse(product_id)
-        if product.type == 'event_booth' and set_qty > 1:
```

Comment:
Should this fix be targeting a stable branch maybe?

Reply:
Indeed we spoke about backporting the fixes with @tde-banana-odoo